blockchain/blockchain-wallet-v4-frontend
# feat(FWN-810): change swap now button

## Description (optional)
### Swap Button Changes
Swap {Asset Name  1} for {Asset Name 2}  → Swap Now 

Screenshot:
![Screen Shot 2021-11-16 at 10 45 19 AM](https://user-images.githubusercontent.com/39777266/142018436-a76b28cd-8775-4df4-8df2-92e891a102d1.png)


## Testing Steps (optional)
Login → Swap → Attempt a Swap → make sure final confirmation button now says `Swap Now`
